Formula/gptfdisk.rb

```diff
@@ -38,15 +36,15 @@ def install
 end
 
 __END__
-diff -ur a/Makefile.mac b/Makefile.mac
---- a/Makefile.mac	2020-02-17 22:34:11.000000000 +0000
-+++ b/Makefile.mac	2020-12-05 22:12:04.000000000 +0000
-@@ -21,7 +21,7 @@
+diff --git a/Makefile.mac b/Makefile.mac
+index ea21fa6..b50bb34 100644
+--- a/Makefile.mac
++++ b/Makefile.mac
+@@ -21,7 +21,7 @@ gdisk:	$(LIB_OBJS) gpttext.o gdisk.o
  #	$(CXX) $(LIB_OBJS) -L/usr/lib -licucore gpttext.o gdisk.o -o gdisk
  
  cgdisk: $(LIB_OBJS) cgdisk.o gptcurses.o
--	$(CXX) $(LIB_OBJS) cgdisk.o gptcurses.o /usr/lib/libncurses.dylib $(LDFLAGS) $(FATBINFLAGS) -o cgdisk
-+	$(CXX) $(LIB_OBJS) cgdisk.o gptcurses.o -L/usr/lib -lncurses $(LDFLAGS) $(FATBINFLAGS) -o cgdisk
+-	$(CXX) $(LIB_OBJS) cgdisk.o gptcurses.o /usr/local/Cellar/ncurses/6.2/lib/libncurses.dylib $(LDFLAGS) -o cgdisk
```

Comment:
Can't this link to the `opt_prefix`?

Reply:
Both the previous diff (used in v1.0.6) and new diff here (v1.0.7) are patching Makefile to use macOS `ncurses`, corresponding to the `uses_from_macos "ncurses"`

In this case, the diff is doing:
`/usr/local/Cellar/ncurses/6.2/lib/libncurses.dylib` ==> `-L/usr/lib -lncurses`

The alternative would be to use Homebrew's `ncurses` along with corresponding path.

Reply:
Damned, misread the patch. `+ -` at the start of the line is confusing